Makefile

```diff
@@ -105,7 +113,9 @@ verify-envoy-protos:
 # as this code is no longer used
 .PHONY: test
 test:
-	PATH=$(DEPSGOBIN):$$PATH ginkgo -r  -v -race -p -tags solokit -compilers=2 -skip multicluster -regexScansFilePath
+ifneq ($(RELEASE), "true")
+	PATH=$(DEPSGOBIN):$$PATH ginkgo -r  -v -race -tags solokit -compilers=2 -skip multicluster -regexScansFilePath -randomizeAllSpecs -randomizeSuites $(TEST_PKG)
+endif
```

Comment:
won't this cause tests to no longer run in ci?

Reply:
Correct, this will cause tests to run if RELEASE != true.

This way, it will run for PRs, but not for releases. This is what we do on our main branch: https://github.com/solo-io/solo-kit/blob/master/Makefile#L123, as tests can be flaky and they dont' produce a different result than PRs